### DISCUSSION TITLE
HMS page in Device tab always blank

### DISCUSSION THREAD

**Discussion by lin-ycv** [OrcaSlicer/OrcaSlicer#7786] [Q&A]
Not sure if this is the intended behavior, but the HMS page should display hms errors? Whenever I encounter an error, this page never shows anything.

**Comment by hexsmith** [comment]
I've seen it populate when the AMS generates an error.  Honestly, the only way I've found to dismiss the errors is to go into the Device/HMS tab and acknowledge them.
